bedrock/settings/static_media.py

```diff
@@ -1130,6 +1152,25 @@
         ),
         'output_filename': 'js/firefox_new-bundle.js',
     },
+    'firefox_new_common': {
+        'source_filenames': (
+            'js/base/search-params.js',
```

Comment:
Hm, no it is not. And now that `common.js` is so small, we may not need that file, either...which would mean getting rid of the `common` bundle altogether. Working on a refactor.